src/elements/content-sidebar/activity-feed/comment/RepliesToggle.js

```diff
@@ -0,0 +1,50 @@
+// @flow
+import * as React from 'react';
+import { FormattedMessage } from 'react-intl';
+
+import messages from './messages';
+import PlainButton from '../../../../components/plain-button';
+
+import './RepliesToggle.scss';
+
+type Props = {
+    isAlwaysExpanded?: boolean,
+    isRepliesLoading: boolean,
+    onHideReplies: (index: number) => void,
+    onShowReplies: () => void,
+    repliesShownCount: number,
+    repliesTotalCount: number,
+};
+
+const RepliesToggle = ({
+    isAlwaysExpanded = false,
+    isRepliesLoading,
+    onShowReplies,
+    onHideReplies,
+    repliesShownCount,
+    repliesTotalCount,
+}: Props) => {
+    if (isAlwaysExpanded || isRepliesLoading || repliesTotalCount <= 1) {
```

Comment:
Pull logic out into separate variable for readability?

Reply:
Don't think it would improve readability in this case since each variable has its own reason to not render the replies component.

Reply:
Now that you say it, I think I can remove isRepliesLoading since it's essentially the same type of prop as isAlwaysExpanded. We can just have the parent determine the rendering of the toggle based on the loading.